data/assets/examples/renderable/renderablenodearrow/renderablenodearrow.asset

```diff
@@ -0,0 +1,39 @@
+-- This example shows an arrow pointing from one scene graph node in the direction of
+-- another. Here, it points from Earth to Mars.
+--
+-- Note that the arrows are generated as objects in 3D space  and need to have a size
+-- that is suitable for their 3D context. Here it is set based on the size of the Earth.
```

Comment:
```suggestion
-- that is suitable for the scene graph nodes they refer to. Here it is set based on the size of the Earth.
```

Reply:
I was a little confused first what the 3D context was, suggestion to change the phrasing